metapackages/plugins-node-all/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@opentelemetry/plugins-node-all",
+  "version": "0.10.2",
+  "description": "Metapackage which bundles opentelemetry node core and contrib plugins",
+  "author": "OpenTelemetry Authors",
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "license": "Apache-2.0",
+  "publishConfig": {
+      "access": "public"
+  },
+  "repository": {
+      "type": "git",
+      "url": "git+https://github.com/open-telemetry/opentelemetry-js.git"
+  },
+  "bugs": {
+      "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+  },
+  "dependencies": {
+      "@opentelemetry/plugin-express": "0.9.0",
+      "@opentelemetry/plugin-ioredis": "0.9.0",
+      "@opentelemetry/plugin-mongodb": "0.9.0",
+      "@opentelemetry/plugin-mysql": "0.9.0",
+      "@opentelemetry/plugin-pg": "0.9.0",
+      "@opentelemetry/plugin-pg-pool": "0.9.0",
+      "@opentelemetry/plugin-redis": "0.9.0",
+      "@opentelemetry/plugins-node-core": "^0.10.2"
```

Comment:
It is my understanding this gets tweaked on release, so I did not touch this area to deal with the current issue. I assume this will get resolved when we release/publish. Let me know if that is not the right approach.